Tests/test_PDB.py

```diff
@@ -1157,6 +1150,48 @@ def test_transform(self):
             for i in range(0, 3):
                 self.assertAlmostEqual(newpos[i], newpos_check[i])
 
+    def test_Vector(self):
+        """Test Vector object"""
+        v1 = Vector(0, 0, 1)
+        v2 = Vector(0, 0, 0)
+        v3 = Vector(0, 1, 0)
+        v4 = Vector(1, 1, 0)
+
+        self.assertEqual(calc_angle(v1, v2, v3), 1.5707963267948966)
+        self.assertEqual(calc_dihedral(v1, v2, v3, v4), 1.5707963267948966)
+        ref = refmat(v1, v3)
+        rot = rotmat(v1, v3)
+        self.assertEqual(ref[0].all(), numpy.array([1.0, 0.0, 0.0]).all())
```

Comment:
I think you should remove every ``.all()`` in your tests, it just reduces an array to a single boolean value (usually ``True`` but it would be ``False`` for a zero vector).

```
$ python3
Python 3.6.0 (v3.6.0:41df79263a11, Dec 22 2016, 17:23:13) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
>>> a = numpy.random.random(3)
>>> b = numpy.random.random(3)
>>> a == b#
array([False, False, False], dtype=bool)
>>> a.all() == b.all()
True
>>> a.all()
True
>>> b.all()
True
```